.github/workflows/pytest-windows.yaml

```diff
@@ -0,0 +1,38 @@
+name: pytest on Windows
+
+on:
+  pull_request:
+  push:
+    branches:
+      - develop
+      - main
+
+jobs:
+  pytest:
+    name: pytest
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+      - name: Get full Python version
+        id: full-python-version
+        shell: bash
+        run: echo ::set-output name=version::$(python -c "import sys; print('-'.join(str(v) for v in sys.version_info))")
+      - name: Install dependencies with conda
+        run: |
+          C:\Miniconda\condabin\conda.bat env update --file environment.yml --name base
```

Comment:
installing packages from environment.yml takes too long for a Github action.
@domfournier, could you have another file (e.g. `environment-test.yml`) which lists only the required packages for tests?

Reply:
hmmm might be tough since the apps depend on so many libraries... not sure what we would be testing without'em

Reply:
I would assume unit tests do not require notebook, ipy modules  nor matplotlib or plotly

Reply:
fair enough, let me see where I can cut

Reply:
actually that's tricky since we import those libraries in the __init__